### PR TITLE
chore: added dependabot.yml

### PR DESCRIPTION
**Pull request by Two-Hearts**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```
Comment (shizhMSFT) [inline .github/dependabot.yml:16]: This repo does not have any go code. Should it be something for npm?

Comment (Two-Hearts) [reply .github/dependabot.yml:16]: oops, nice catch!

Comment (Two-Hearts) [reply .github/dependabot.yml]: updated

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**